ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -249,12 +250,15 @@ message DamlConfigurationSubmission {
   // Implementers are free to select adequate mechanism e.g. UUID or similar.
   string submission_id = 1;
 
+  // Submitting participant's id.
+  string participant_id = 2;
```

Comment:
Changing the type and semantics of protobuf fields is a dangerous habit to get into.

Reply:
100% agree in the general case. Here I'm making a conscientious decision to break it as there has never existed a mechanism to submit these type of messages and there are no ledgers out there containing this data. In this case I think this is reasonable, but happy to have a discussion if you disagree and find an alternative.

Reply:
The alternative would be to use `string participant_id = 4;`.
I'm more concerned about reeeeaaaaallly getting into the habit of just never doing it. If this has never been persisted before, then fine. Let's not spend any more time on it.